src/libraries/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.Aead.cs

```diff
@@ -168,66 +164,55 @@ internal static partial class AppleCrypto
 
         [LibraryImport(Libraries.AppleCryptoNative)]
         [UnmanagedCallConv(CallConvs = [ typeof(CallConvSwift) ])]
-        private static unsafe partial int AppleCryptoNative_ChaCha20Poly1305Encrypt(
-            byte* keyPtr,
-            int keyLength,
-            byte* noncePtr,
-            int nonceLength,
-            byte* plaintextPtr,
-            int plaintextLength,
-            byte* ciphertextPtr,
-            int ciphertextLength,
-            byte* tagPtr,
-            int tagLength,
-            byte* aadPtr,
-            int aadLength);
+        private static unsafe partial void AppleCryptoNative_ChaCha20Poly1305Encrypt(
+            UnsafeBufferPointer<byte> key,
+            UnsafeBufferPointer<byte> nonce,
+            UnsafeBufferPointer<byte> plaintext,
+            UnsafeMutableBufferPointer<byte> ciphertext,
+            UnsafeMutableBufferPointer<byte> tag,
+            UnsafeBufferPointer<byte> aad,
+            out SwiftError error
+        );
```

Comment:
The closing paren should be on the same line as `error`. Same feedback for the other library imports.

```suggestion
            out SwiftError error);
```